src/include/homestore/btree/detail/btree_mutate_impl.ipp

```diff
@@ -135,10 +135,10 @@ retry:
             if (child_node->is_leaf()) {
                 // We get the trimmed range only for leaf because this is where we will be inserting keys. In
                 // interior nodes, keys are always propogated from the lower nodes.
-                bool is_inp_key_lesser;
-                req.trim_working_range(
-                    my_node->min_of(s_cast< const K& >(req.input_range().end_key()), curr_idx, is_inp_key_lesser),
-                    is_inp_key_lesser ? req.input_range().is_end_inclusive() : true);
+                bool is_inp_key_lesser = false;
+                K end_key = my_node->min_of(s_cast< const K& >(req.input_range().end_key()), curr_idx, is_inp_key_lesser);
+                bool end_incl = is_inp_key_lesser ? req.input_range().is_end_inclusive() : true;
```

Comment:
Is there a logic change here, sounds like you are storing in a variable and then passing it?

Reply:
No logic change. Just separated into multiple lines for readability.